src/VMware/custom/New-AzVMwarePrivateCloud.ps1

```diff
@@ -185,12 +190,35 @@ function New-AzVMwarePrivateCloud {
     )
     
     process {
+
+        if(!$PSBoundParameters.ContainsKey('AcceptEULA')){
+            try {
+                $legalTermPath = Join-Path $PSScriptRoot -ChildPath "LegalTerm.txt"
+                $legalTerm = (Get-Content -Path $legalTermPath) -join "`r`n"
+
+                $confirmation = Read-Host $legalTerm"`n[Y] Yes  [N] No  (default is `"N`")"
```

Comment:
Could you add an example on your PR description?

Reply:
something like unable to read the legal term file